tool/gravity/cli/operation.go

```diff
@@ -74,6 +76,61 @@ func executePhase(localEnv *localenv.LocalEnvironment, environ LocalEnvironmentF
 	}
 }
 
+const (
+	// planRollbackWarning is shown when "gravity rollback" command is launched
+	// without --confirm flag.
+	planRollbackWarning = `You are about to rollback the following operation:
+
+%v
+Consider checking the operation plan and using --dry-run flag first to see which actions will be performed.
+You can suppress this warning in future by providing --confirm flag.
+
+`
+	// unsupportedRollbackWarning is shown for operations that "gravity rollback"
+	// command does not support.
+	unsupportedRollbackWarning = `"gravity rollback" command does not support %q operations currently.
```

Comment:
~~currently~~
`Operation %v does not support automatic rollback.`
or
`Operation %v does not support full plan rollback.`